docs/v0.4.0/_src/tutorials/tutorials/1.md

```diff
@@ -188,7 +188,7 @@ reader = FARMReader(model_name_or_path="deepset/roberta-base-squad2", use_gpu=Fa
 
 ```python
 # Alternative:
-# reader = TransformersReader(model="distilbert-base-uncased-distilled-squad", tokenizer="distilbert-base-uncased", use_gpu=-1)
+# reader = TransformersReader(model_name_or_path="distilbert-base-uncased-distilled-squad", tokenizer="distilbert-base-uncased", use_gpu=-1)
```

Comment:
We should not change the docs for version 0.4.0 (as the parameter was there still "model")

Reply:
Reverted changes from all 0.4.0 files